patches/tModLoader/Terraria/ModLoader/RecipeEditor.cs

```diff
@@ -28,17 +28,17 @@ public class RecipeEditor
 			if (itemID <= 0 || itemID >= ItemLoader.ItemCount) {
 				throw new RecipeException("No item has ID " + itemID);
 			}
-			for (int k = 0; k < Recipe.maxRequirements; k++) {
-				if (recipe.requiredItem[k].type == 0) {
-					recipe.requiredItem[k].SetDefaults(itemID, false);
-					recipe.requiredItem[k].stack = stack;
-					return;
-				}
-				if (recipe.requiredItem[k].type == itemID) {
-					recipe.requiredItem[k].stack += stack;
-					return;
-				}
-			}
+			// for (int k = 0; k < Recipe.maxRequirements; k++) {
+			// 	if (recipe.requiredItem[k].type == 0) {
+			// 		recipe.requiredItem[k].SetDefaults(itemID, false);
```

Comment:
Use comment blocks kthx

Reply:
RecipeEditor will get removed so I'm not worried about this